### ISSUE TITLE
[Bug]: Module not found: Can't resolve '@rvf/set-get'

### ISSUE THREAD

**Issue by markedwards** [airjp73/rvf#413]
### Which packages are impacted?

- [ ] `@rvf/react`
- [ ] `@rvf/react-router`
- [ ] `@rvf/zod`
- [ ] `@rvf/yup`
- [X] `zod-form-data`

### What version of these packages are you using?

- `zod-form-data`: 2.0.4

### Please provide a link to a minimal reproduction of the issue.

N/A

### Steps to Reproduce the Bug or Issue

1. Install as per documentation `yarn add zod-form-data`
2. Observe `Module not found: Can't resolve '@rvf/set-get'` at runtime

### Expected behavior

If it is expected to install `@rvf/set-get`, shouldn't it be stated in the README for `zod-form-data`?

### Platform

- OS: Linux
- Browser: Chrome
- Version: 131.0.6778.205

**Comment by airjp73** [comment]
Yeah, that's definitely a mistake. I just published 2.0.5 with a fix. Please give it a try and let me know if that works for you! :)

**Comment by airjp73** [comment]
Going to close this since it's addressed in 2.0.5. Please feel free to re-open if it still doesn't work! :)

**Comment by markedwards** [comment]
Confirmed all good.  Thanks!
